### PR TITLE
[docs] add manual doc in Chinese

### PR DESCRIPTION
**Pull request by awusan125**
resolve https://github.com/secretflow/scql/issues/66

### CODE REVIEW

**Comment by awusan125** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by awusan125** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po [hunk 1]
```diff
@@ -21,97 +21,102 @@ msgstr ""
...
 #: ../../reference/scql_manual.rst:34
 msgid "CREATE/DROP USER Statement"
-msgstr ""
+msgstr "CREATE/DROP USER 语句"
 
 #: ../../reference/scql_manual.rst:46 ../../reference/scql_manual.rst:69
 #: ../../reference/scql_manual.rst:95 ../../reference/scql_manual.rst:130
 msgid "Examples:"
-msgstr ""
+msgstr "例子："
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po:111]: 例子 - > 样例

File: docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po [hunk 2]
```diff
@@ -120,40 +125,44 @@ msgid ""
 " user runs queries on virtual table, which helps simplify privilege "
 "control and usage."
 msgstr ""
+"此处创建表意味着将 'tbl_name' 映射到现有的物理表，"
+"该物理表由 'TID' 指定且位于相关方上。在 SCQL 中，"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po:129]: 相关方 -> 相关参与方

File: docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po [hunk 3]
```diff
@@ -120,40 +125,44 @@ msgid ""
...
+"用户在虚拟表上运行查询可以有助于简化特权"
+"控制和使用。"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po:131]: 特权 -> 权限

File: docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po [hunk 4]
```diff
@@ -120,40 +125,44 @@ msgid ""
...
 
 #: ../../reference/scql_manual.rst:230
 msgid "Todo"
-msgstr ""
+msgstr "待办"
 
 #: ../../reference/scql_manual.rst:230
 msgid "this part is not ready, please check later"
-msgstr ""
+msgstr "该部分尚未准备好，请稍后查看"
```
Comment (Mingbo-Lee) [inline docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po:168]: 稍后 -> 之后

**Review by Mingbo-Lee — CHANGES_REQUESTED** [review]

**Review by Mingbo-Lee — APPROVED** [review]
LGTM

File: docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po [hunk 5]
```diff
@@ -21,97 +21,102 @@ msgstr ""
...
-msgstr ""
+msgstr "SCQL 语句"
 
 #: ../../reference/scql_manual.rst:31
 msgid ""
 "SCQL is compatible with most MySQL syntax, which makes it easy to use. "
 "For syntax differences between SCQL and MySQL, please read "
 ":doc:`mysql_compatibility`."
 msgstr ""
+"SCQL 兼容大多数MySQL语法，这使得它易于使用。"
```
Comment (tongke6) [inline docs/locales/zh_CN/LC_MESSAGES/reference/scql_manual.po:100]: MySQL 前后请添加相应的空格

**Review by tongke6 — APPROVED** [review]
LGTM

**Pull request merged**